docs/src/man/structure.md

```diff
@@ -227,6 +227,10 @@ Various functions are provided to calculate spatial quantities for proteins:
 | `rmsd`               | RMSD between two elements of the same size - assumes they are superimposed                      |
 | `displacements`      | `Vector` of displacements between two elements of the same size - assumes they are superimposed |
 
+The omega, phi and psi angle functions can take either a pair of residues or a chain and a position. Omega and phi will
```

Comment:
Also given that you are saying this here, perhaps change the entries in the table above to be e.g. "Omega dihedral angle" for `omegaangle`.